tests/vm_scheduling/env_config.yml

```diff
@@ -0,0 +1,8 @@
+env:
+  scenario: vm_scheduling
+  topology: azure.2019.10k
+  start_tick: 1
+  durations: 8638
+  resolution: 1
```

Comment:
Also, besides the topology problem. Since it is also for a specific test, the configs in this file can also be written down in the code directly.